components/esp-tls/Kconfig

```diff
@@ -115,4 +115,12 @@ menu "ESP-TLS"
         help
             Enable detailed debug prints for wolfSSL SSL library.
 
+    config ESP_TLS_OCSP_CHECKALL
+        bool "Enabled full OCSP checks for wolfSSL"
```

Comment:
Im sorry, is it possible to change this wolfSSL to ESP-TLS as well?
I missed this earlier